### DISCUSSION TITLE
VS Code & Rider AutoComplete - HighLight and UWP Differences

### DISCUSSION THREAD

**Discussion by alperokur** [unoplatform/uno#11414] [Q&A]
Hello, I want to develop a Linux environment UWP application. Any chance of doing this successfully? I tried using Visual Studio Code and Rider programs but I can't use XAML autocomplete and highlighting. My second question is, is there a resource that explains the difference between UWP and UWP-NET6? Which one should I choose?

![Ekran Görüntüsü - 2023-02-21 13-00-21](https://user-images.githubusercontent.com/36308747/220544463-7295dce0-a0b3-4231-895b-09691af56cce.png)

![Ekran Görüntüsü - 2023-02-21 13-02-08](https://user-images.githubusercontent.com/36308747/220544491-c20b3087-d427-42be-82c0-8ee6a1bb49f5.png)

**Comment by jeromelaban** [comment]
Thanks for the feedback. Developing with the UWP API in VS Code is supported, though based on your screenshot, you do not have selected the "active" project. 

This is caused by Omnisharp (C# Support for VS Cod) that failed to initialize (the little flame at the bottom left). You should be seeing error messages in the `Omnisharp Log` output tab that you will need to fix first. Once fixed, you'll be seeing the name of the solution instead of the flame icon, and you'll be able to select a single project that will enable autocomplete.
